### PR TITLE
Improve Transcode Quality

### PR DESCRIPTION
**Pull request by leszko**
Related to https://github.com/livepeer/lpms/pull/371

### CODE REVIEW

File: net/lp_rpc.proto [hunk 1]
```diff
@@ -301,6 +301,7 @@ message VideoProfile {
     CHROMA_444 = 2;
   }
   ChromaSubsampling chromaFormat = 27;
+  uint32 crf = 28;
```
Comment (emranemran) [inline net/lp_rpc.proto]: Is this the default? I'd suggest setting this lower since it translates to a pretty high CQ.

Comment (emranemran) [reply net/lp_rpc.proto]: OK i see now after reading your comment in the other PR. But should this be 27 like the other PR?

Comment (leszko) [reply net/lp_rpc.proto]: This is not a value, just the field number in the probuf. So, in short, in just needs to be the next number after the line above.

**Comment by leszko** [comment]
@emranemran as discussed, I renamed `crf` to `quality` everywhere. PTAL.

**Review by thomshutt — APPROVED** [review]

**Pull request merged**